vantage6/common/docker_addons.py

```diff
@@ -34,7 +35,7 @@ def check_docker_running():
     """ Return True if docker engine is running"""
     try:
         docker_client.ping()
-    except Exception as e:
+    except Exception:
```

Comment:
lets catch `e` and print it. 
`log.debug(e)`

Reply:
Changed accordingly